ci/README.md

```diff
@@ -32,9 +32,6 @@ Any file or directory in this subdirectory should be documented here.
 9. Update the AUR package.
```

Comment:
I can't comment on unchanged lines that are far away - but step 7 in this file is no longer necessary now that we moved from `hub` to `gh`.